.github/workflows/coverage.yml

```diff
@@ -44,4 +44,4 @@ jobs:
 
       - name: Upload coverage to Codecov
         run: |
-          bash <(curl -s https://codecov.io/bash) -t ${{ secrets.CODECOV_TOKEN }}
+          bash <(curl -s https://codecov.io/bash) || echo 'Codecov failed to upload'
```

Comment:
we don't need to echo this and in fact it might give us a false positive if an upload failed